src/opencl/image_function_opencl.cpp

```diff
@@ -292,6 +293,25 @@ namespace
             }
         }
 
+        __kernel void setPixelOpenCL( __global const uchar * data, uint offset, uint rowSize, uint width, uint height, uint value )
+        {
+            const size_t x = get_global_id(0);
+            const size_t y = get_global_id(1);
+
+            if ( x < width && y < height ) {
+                in[offset + y * rowSize + x] = value
+            }
+        }
+
+        __kernel void setPixelOpenCL( __global const uchar * data, volatile __global uint * offsets, uint rowSize, uint width, uint height,
+                                      volatile __global uint * pointX,  volatile __global uint * pointY, uint pointSize, uint value)
```

Comment:
You don't need volatile., variables are not going to change make them `__global const uint * pointX,  __global const uint * pointY`

Reply:
noted and changed, I thought volatile was to denote an array of some sorts. :)